src/scripts.js

```diff
@@ -153,9 +153,8 @@ function findAvailableRooms() {
         }).map(booking => booking.roomNumber)
 
     let availableRooms = hotel.filter(room => !unBooked.includes(room.number))
-    console.log(availableRooms)
 
-    availableRooms = findRoomsByType(availableRooms, typeFilter.value)
+    availableRooms = findRoomsByType(availableRooms)
     
```

Comment:
Are you calling findRoomsByType inside findRoomsByType? Maybe this is the problem? I've looked at it for too long now LOL